packages/nouns-contracts/contracts/NounsAuctionHouseV3.sol

```diff
@@ -267,27 +320,36 @@ contract NounsAuctionHouseV2 is
 
     /**
      * @notice Settle an auction, finalizing the bid and paying out to the owner.
+     *   Part of the winning bid amount is sent immediately to the owner, and the rest is streamed via `streamEscrow`.
```

Comment:
nit: extra whitespace before `Part` seems on accident?

Reply:
actually on purpose, because I see it as part of the `@notice` paragraph. I think we've formatted like this in the past. but I don't care so much